### ISSUE TITLE
Conversion for `LaurentPolynomial` looses `m`

### ISSUE THREAD

**Issue by martinholters** [JuliaMath/Polynomials.jl#287]
```
julia> p = LaurentPolynomial([1], -5)
LaurentPolynomial(x⁻⁵)

julia> convert(LaurentPolynomial{Float64}, p)
LaurentPolynomial(1.0)
```
I was expecting `LaurentPolynomial(1.0*x⁻⁵)`.

**Comment by jverzani** [comment]
Ahh, this needs another `convert` method. Thanks. Will address.

**Comment by martinholters** [comment]
I've added this one which seems to do the trick:
```julia
Base.convert(::Type{T}, p::LaurentPolynomial) where {T<:LaurentPolynomial} = T(p.coeffs, p.m[], p.var)
```

**Comment by jverzani** [comment]
Thanks. I tried a few other ways to do this, and was getting issues. Do you want to put in a PR, or should I? If you do, can you bump the version number?

**Comment by martinholters** [comment]
Just go ahead, I'm calling it a day now.

**Comment by martinholters** [comment]
Fixed by #289.
